src/patternfly/components/Nav/nav.scss

```diff
@@ -465,13 +467,13 @@
   border: 0;
 
   // Add transition to base
-  & > * {
-    transition: var(--pf-c-nav__list-toggle--Transition);
+  > .pf-c-nav__toggle-icon {
+    transition: var(--pf-c-nav__list-toggle--icon--Transition);
   }
 }
 
 // Rotate the icon
-.pf-c-nav__item.pf-m-expanded .pf-c-nav__toggle > * {
+.pf-c-nav__item.pf-m-expanded .pf-c-nav__toggle > .pf-c-nav__toggle-icon {
   transform: var(--pf-c-nav__item--m-expanded__toggle-icon--Transform);
```

Comment:
is there a need for both `nav__toggle` and `nav__toggle-icon`?

Reply:
good point